sdk/python/feast/batch_feature_view.py

```diff
@@ -0,0 +1,61 @@
+from datetime import timedelta
+from typing import Dict, List, Optional, Union
+
+from google.protobuf.duration_pb2 import Duration
+
+from feast import Entity, Feature, Field
```

Comment:
nit: elsewhere we tend to do `from feast.entity import Entity` etc.

we should try to stay consistent

Reply:
good catch